src/index.html

```diff
@@ -7,12 +7,53 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
-      href="./styles/index.scss"
+    />
+    <link
+      rel="/src/styles/index.scss"
+      href="/src/styles/index.css"
     />
   </head>
+
   <body>
     <h1>Product cards</h1>
+    <div
+      data-qa="card"
+      class="card"
+    >
+      <div class="card__image">
+        <img
+          src="./images/imac.jpeg"
```

Comment:
The 'alt' attribute should accurately describe the image content. If the image is of a MacBook, then the 'alt' text is fine. However, if the image is of an iMac, the 'alt' text should be updated to reflect that.